src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -53,20 +53,22 @@
      (cond
        (not supported-file?)
        [fallback-view
-        {:label   (i18n/label :t/unsupported-file)
-         :counter counter
-         :theme   theme}]
+        {:label    (i18n/label :t/unsupported-file)
+         :counter  counter
+         :theme    theme
+         :on-mount on-collectible-load}]
 
        image-error?
        [fallback-view
-        {:label   (i18n/label :t/cant-fetch-info)
-         :counter counter
-         :theme   theme}]
+        {:label    (i18n/label :t/cant-fetch-info)
+         :counter  counter
+         :theme    theme
+         :on-mount on-collectible-load}]
```

Comment:
When the collectible is unsupported or has an error, we don't use `rn/image` so the `on-collectible-load` callback responsible of meassuring the dimensions taken by the collectible is not called.

Now we call it `on-mount` of the fallback view.

cc: @ajayesivan

Reply:
Make sense 👍